docs/api_output.rst

```diff
@@ -35,7 +35,7 @@ RGBLED
 ======
 
 .. autoclass:: RGBLED(red, green, blue, active_high=True)
-    :members: on, off, red, green, blue, value
+    :members: on, off, blink, red, green, blue, value
```

Comment:
If we have `on` and `off`, should we also have `toggle` ?
And should we also have `RGBLED.is_lit` ?
Does the `color` property need to be mentioned here?


Definitely feels like there's an implicit "LED interface" shared between LED, PWMLED and RGBLED... any (easy) way that implicit interface could be formalised, or is this just an example of duck-typing?